ecommerce_api_client/tests/test_client.py

```diff
@@ -21,10 +23,42 @@ class EcommerceApiClientTests(TestCase):
     """ Tests for the E-Commerce API client. """
 
     @ddt.data(
-        {'url': URL, 'signing_key': SIGNING_KEY, 'username': USERNAME, 'full_name': FULL_NAME, 'email': EMAIL},
```

Comment:
Also, you don't seem to be actually testing that the values are set in the JWT. That's the entire point of the test.